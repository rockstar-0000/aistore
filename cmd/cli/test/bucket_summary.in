ais bucket summary ais://tmp // FAIL "bucket "ais://tmp" does not exist"
ais bucket create ais://$BUCKET_1 | awk '{print $1 " " $2}'
ais bucket create ais://$BUCKET_2 | awk '{print $1 " " $2}'
ais bucket summary ais://$BUCKET_1
ais bucket summary aiss://$BUCKET_1 // FAIL "invalid backend provider "aiss""

ais archive gen-shards "ais://$BUCKET_1/tmp/shard-{0..149}.tar" --fcount 1 --fsize 1KB --conc 1 --cleanup // IGNORE
ais archive gen-shards "ais://$BUCKET_2/tmp/shard-{0..19}.tar" --fcount 10 --fsize 1KB --conc 1 --cleanup // IGNORE
ais bucket summary ais://$BUCKET_1
sleep 1
ais bucket summary ais://$BUCKET_2
sleep 1
