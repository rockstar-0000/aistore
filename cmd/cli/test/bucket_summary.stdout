"ais://$BUCKET_1" created
"ais://$BUCKET_2" created
NAME             OBJECTS (cached, remote)    OBJECT SIZES (min, avg, max)    TOTAL OBJECT SIZE (cached, remote)    USAGE(%)
ais://$BUCKET_1  0 0                         0B    0B    0B                  0B  0B                                0%
NAME             OBJECTS (cached, remote)    OBJECT SIZES (min, avg, max)    TOTAL OBJECT SIZE (cached, remote)    USAGE(%)
ais://$BUCKET_1  150 0                       2.50KiB  2.50KiB   2.50KiB      375.00KiB  0B                         0%
NAME             OBJECTS (cached, remote)    OBJECT SIZES (min, avg, max)    TOTAL OBJECT SIZE (cached, remote)    USAGE(%)
^ais://$BUCKET_2  20 0.*$
